data/l10n/bzflag_sk.po

```diff
@@ -0,0 +1,3205 @@
+# Slovak translation of BZFlag
+
+msgid "xx"
+msgstr "Zaklad prekladu"
+
+msgid "translator"
+msgstr "Jose Riha"
+
+#############
+## GENERAL ##
+#############
+
+msgid "Off"
+msgstr "Vypnute"
+
+msgid "On"
+msgstr "Zapnute"
+
+msgid "Unavailable"
+msgstr "Nedostupne"
+
+msgid "None"
+msgstr "Ziadne"
+
+msgid "Fast"
+msgstr "Rychlo"
+
+msgid "Best"
+msgstr "Najlepsie"
+
+msgid "NOTE: "
+msgstr "POZNAMKA: "
+
+msgid "Page {1} of {2}"
+msgstr "Strana {1} z {2}"
+
+msgid "Auto"
+msgstr "Auto"
+
+msgid "Pause/Resume:"
+msgstr "Pauza/Pokracovat:"
+
+msgid "Slow Keyboard Motion:"
+msgstr "Jemny pohyb klavesnicou:"
+
+msgid "<not mapped>"
+msgstr "<nenamapovane>"
+
+msgid "No"
+msgstr "Nie"
+
+msgid "Yes"
+msgstr "Ano"
+
+msgid "N/A"
+msgstr "Nedost"
+
+msgid "Failed... unknown error (%d)."
+msgstr "Zlyhalo... neznama chyba (%d)."
+
+msgid "Silenced All Msgs"
+msgstr "Umlcat vsetky spravy"
+
+msgid " or "
+msgstr " alebo "
+
+##################
+## INPUT EVENTS ##
+##################
+
+msgid "Pause"
+msgstr "Pause"
+
+msgid "Home"
+msgstr "Home"
+
+msgid "End"
+msgstr "End"
+
+msgid "Left Arrow"
+msgstr "Sipka vlavo"
+
+msgid "Right Arrow"
+msgstr "Sipka vpravo"
+
+msgid "Up Arrow"
+msgstr "Sipka hore"
+
+msgid "Down Arrow"
+msgstr "Sipka dole"
+
+msgid "Page Up"
+msgstr "Page Up"
+
+msgid "Page Down"
+msgstr "Page Down"
+
+msgid "Insert"
+msgstr "Insert"
+
+msgid "Backspace"
+msgstr "Backspace"
+
+msgid "Delete"
+msgstr "Delete"
+
+msgid "Kp0"
+msgstr "Kp0"
+
+msgid "Kp1"
+msgstr "Kp1"
+
+msgid "Kp2"
+msgstr "Kp2"
+
+msgid "Kp3"
+msgstr "Kp3"
+
+msgid "Kp4"
+msgstr "Kp4"
+
+msgid "Kp5"
+msgstr "Kp5"
+
+msgid "Kp6"
+msgstr "Kp6"
+
+msgid "Kp7"
+msgstr "Kp7"
+
+msgid "Kp8"
+msgstr "Kp8"
+
+msgid "Kp9"
+msgstr "Kp9"
+
+msgid "Kp_Period"
+msgstr "Kp_Period"
+
+msgid "Kp_Divide"
+msgstr "Kp_Lomka"
+
+msgid "Kp_Multiply"
+msgstr "Kp_Hviezdicka"
+
+msgid "Kp_Minus"
+msgstr "Kp_Minus"
+
+msgid "Kp_Plus"
+msgstr "Kp_Plus"
+
+msgid "Kp_Enter"
+msgstr "Kp_Enter"
+
+msgid "Kp_Equals"
+msgstr "Kp_RovnaSa"
+
+msgid "F1"
+msgstr "F1"
+
+msgid "F2"
+msgstr "F2"
+
+msgid "F3"
+msgstr "F3"
+
+msgid "F4"
+msgstr "F4"
+
+msgid "F5"
+msgstr "F5"
+
+msgid "F6"
+msgstr "F6"
+
+msgid "F7"
+msgstr "F7"
+
+msgid "F8"
+msgstr "F8"
+
+msgid "F9"
+msgstr "F9"
+
+msgid "F10"
+msgstr "F10"
+
+msgid "F11"
+msgstr "F11"
+
+msgid "F12"
+msgstr "F12"
+
+msgid "Help"
+msgstr "Pomoc"
+
+msgid "Print"
+msgstr "Print"
+
+msgid "Sysreq"
+msgstr "Sysreq"
+
+msgid "Break"
+msgstr "Break"
+
+msgid "Menu"
+msgstr "Menu"
+
+msgid "Power"
+msgstr "Power"
+
+msgid "Euro"
+msgstr "Euro"
+
+msgid "Undo"
+msgstr "Undo"
+
+msgid "Left Mouse"
+msgstr "Lave tlacidlo mys"
```

Comment:
I noticed that for "Left Mouse", the translation used "mys" while "Middle Mouse" and "Right Mouse" used "mysi".  I'm unsure if that's a typo.

Reply:
great observation - yes, there is (or was) a typo. now fixed. thanks!